eng/build.sh

```diff
@@ -116,23 +117,46 @@ while [[ $# > 0 ]]; do
         shift 2
       fi
       ;;
+
      -arch)
       if [ -z ${2+x} ]; then 
         echo "No architecture supplied. See help (--help) for supported architectures." 1>&2
         exit 1
       fi
-      arch=$2
+      passedArch="$(echo "$2" | awk '{print tolower($0)}')"
+      case "$passedArch" in
+        x64|x86|arm|armel|arm64|wasm)
```

Comment:
Is it intentional that <code>armel</code> is missing in <code>build.ps1</code> above?

Reply:
I think it is, as there is no armel flavor of Windows.

Reply:
Yes, I followed the `help` command when adding these validations. Thanks for the clarification Jan!